reference/resources/bgppeer.md

```diff
@@ -47,6 +47,7 @@ spec:
 | keepOriginalNextHop | Maintain and forward the original next hop BGP route attribute to a specific Peer within a different AS. | | boolean |
 | password   | BGP password for the peerings generated by this BGPPeer resource. |  | [BGPPassword](#bgppassword) | `nil` (no password) |
 | sourceAddress  | Specifies whether and how to configure a source address for the peerings generated by this BGPPeer resource.  Default value "UseNodeIP" means to configure the node IP as the source address.  "None" means not to configure a source address. | "UseNodeIP", "None"  | string | "UseNodeIP" |
+| maxRestartTime  | Restart time that is announced by BIRD in the BGP graceful restart capability and that specifies how long the neighbor would wait for the BGP session to re-establish after a restart before deleting stale routes. Note: extra care should be taken when changing this configuration, as it may break networking in your cluster. When not specified, BIRD uses the default value of 120 seconds. | `10s`, `120s`, `2m` etc.  | [Duration string][parse-duration] | `nil` (empty config, BIRD will use the default value of `120s`) |
```

Comment:
Suggestions for clarity. Not sure BIRD is necessary to expose.

Time after a BGP session restart for a peer to wait before deleting stale routes. Default: 120 seconds. **Important!** Changing this value may break networking in your cluster, so plan accordingly.

Reply:
Not clear whether we mean updating the default could break the network, or any value change.

Reply:
Thanks, @bmckercher123!

Re: the default, I just wanted to make clear that the calico default is `nil` (which will be omitted), which will result in not having a config in BIRD, which will then result in it using its 120s default (and also I don't know how likely it is, but in case BIRD changes its default some day, to indicate that may be the reason for a change).
In case we are not exposing BIRD, I still think we should at least mention the BGP graceful restart capability, as it is a 
standard BGP feature.

Re: breakage, we don't offer a way of updating the default, only "normal" value changes. Any value change to a "non-sane" value could break the network.

How about:
BGP graceful restart value. Time after a BGP session restart for a peer to wait before deleting stale routes. Default: 120 seconds. **Important!** Updating this value may break networking in your cluster, so care should be taken.

I still wonder if BIRD should be mentioned since it may help diagnosing problems...